### ISSUE TITLE
Permit Acquisition missions

### ISSUE THREAD

**Issue by Darkcyde13** [EDCD/EDDI#2277]
### Expected
EDDI to use the correct spelling in "Permit Acquisition" for all Permit mission related scripts.
EDDI to not count the Permit Acquistion missions as active log entries.

### Observed
EDDI uses "Permit Aquisition" (missing the 'c').
EDDI adds the missions to the Mission Monitor log.

### Steps to reproduce
1. Take a Permit Acquisition mission.
2. Observe the .tags[0] showing it as "Permit Aquisition", and I think the .invariantTags may also be incorrect.
3. Note that the mission is logged to the Mission Monitor.

### Configuration
* Version [Please give full version numbers here, not just "latest" or "current"]
  * EDDI Version:  4.0.1
  * VoiceAttack Version: N/A
* Localization [if applicable]
  * EDDI's language: British English
  * Voice: Ivona 2 Amy

## My Investigation
I've been trying to update and improve my missions routing scripts while working on gaining some in-game permits, and have realised that the word Acquisition has been spelled incorrectly in EDDI.  I've also noted that EDDI counts these missions as active ones in the Mission Monitor log, but the game doesn't, instead they are 'completed' as soon as they are 'accepted'.

### Investigation Notes
While my scripts are customised, they still use the same variables as the defaults ( .tags and .invariantTags) and it's these that are using the incorrect spelling.
https://whichiscorrect.com/acquisition-vs-aquisition/

There are no corresponding 'mission complete' Journal entries for Permit Acquisition missions, as they are not logged in game.  Therefore, the missions stay in the Mission Monitor until the player relogs and the monitor is updated with current missions.  This also means the section in 'Mission completed' is probably redundant, as it unlikely it will ever be used.

Journal event:
`{ "timestamp":"2021-10-20T21:02:15Z", "event":"MissionAccepted", "Faction":"Hodack Prison Colony", "Name":"MISSION_genericPermit1", "LocalisedName":"Permit Acquisition Opportunity", "Wing":false, "Influence":"None", "Reputation":"None", "MissionID":814712321 }`

EDDI script code (from 'Mission accepted'):
EDDI default: `{set type to mission.tags[0]}`
Code from my version:  `{set type to cat(mission.tags[0], " mission")}`

EDDI output:
> Permit Aquisition mission for Hodack Prison Colony accepted.

I think it's also the same for the (now depreciated) '.type'.

Permit Acquisition missions are not part of the player's mission log (at least in Odyssey anyway).  When a permit mission is accepted, the player is simply awarded the permit instantly, with a comms message letting them know.  EDDI still includes these as missions in the Mission Monitor log, which is incorrect.  I know this will be cleared when logging on next time, but until then it is still considered as an active mission.

### EDDI Logs
N/A

### Player journals
N/A

**Comment by Tkael** [comment]
Interesting. So permit unlock missions don't go through the normal progression in Odyssey that we see in Horizons? Or they do but your journal only records the beginning of the mission and not the end? First time I'm hearing about this but it might be related to a more general FDev Odyssey bug where the `MissionCompleted` journal entry can in some cases be omitted.

Can you please provide the complete player journal from your permit acquisition mission?
Do you have any more permits to acquire? I'd like to see how repeatable the omission of `MissionCompleted` is on these kinds of missions.

**Comment by Darkcyde13** [comment]
[Journal.211020163445.01.zip](https://github.com/EDCD/EDDI/files/7420087/Journal.211020163445.01.zip)
Here's my log for that play session.  As I mentioned above, there are no corresponding 'completed' entries.  As soon as the mission is 'accepted' (immediately when you click the accept button), you receive an in-game email saying the faction has issued the permit.

Here's my log for the previous permit I earned.
[Journal.211019200605.01.zip](https://github.com/EDCD/EDDI/files/7420117/Journal.211019200605.01.zip)

I can look back further if you like, for the others I have got over the last few weeks.  However, I'm 99% certain that they are all the same.  My guess is that FDev made this a permanent change, as it is/was pointless to take a mission that you only had to immediately hand back in.  Doing it this way saves time and removes unnecessary clicks.

I do have a couple left to acquire.  Here's an excerpt from my list of all permits, with those left to obtain:
```
...
   Isinor - Chapter of Isinor. Contains a persistent Convoy Beacon orbiting the ELW of Maodun.  Working on... HR 4803
 * Jotun - Dukes of Jotun.
 X Lia Fail - Part of the starter area.
 - LFT 509 - Currently unobtainable.
   LTT 198 - Torval Mining Ltd. Megaship in EGM 559, after faction change (09-10-21).
   Luyten 205-128 - Revolutionary Luyten 205-128 Coalition.
   Luyten 347-14 - Luyten 347-14 Prison Colony.
...
```
The asterisk is one I have already got, the X is one for the starter area (so revoked when you leave), and the hyphen is one that's currently unobtainable.  I am working on the Isinor one, and then only have those three others to get.

**Comment by Tkael** [comment]
It has been a while since I've done a permit acquisition mission, but it believe that I had to complete an objective to complete the missions... they couldn't be handed in instantly. 

Donation missions (which can be completed immediately) still require multiple clicks to accept and then complete too.

**Comment by Darkcyde13** [comment]
Agreed.  I do seem to remember permit missions I had before Odyssey, worked as you describe (as normal missions).

You're right about donation missions too, at least for the donating credits ones.  I don't know anyone who would be silly enough to take a donate credits mission without having more than enough spare cash to complete it immediately. ;)

**Comment by Darkcyde13** [comment]
I've built up my rep to allied with the last factions I needed to gain my final permits, and I'm ready to accept the missions.  What would you like me to do in order to provide the information you require?  Bear in mind that I only need four more, so will only have four opportunities to collect any data for you.

**Comment by Tkael** [comment]
Ok, so these are the questions that I have.
1. I'd like to witness an Odyssey permit acquisition in both Horizons and Odyssey (recorded video perhaps?). Are they the same or are they different now in Horizons and Odyssey?
2. Is this the intended behavior or is this something that FDev would regard as a bug (subject to eventual correction)? Once we've gathered some real-game evidence then we can present that data to FDev and try to get a definitive answer.

**Comment by Darkcyde13** [comment]
OK, so this has been a learning experience with making videos.  I first tried the AMD recording built in to the software, but had unpredictable results.  So, I've tried OBS as I had seen someone mention this in a forum post.  After a few practice recordings, I made one for a permit acceptance.  I've made a short video (45 seconds) of gaining the Isinor permit, but it's too big to post here (it's 26.9MB).  This is a link to it on my Dropbox, let me know if it works or not:
https://www.dropbox.com/s/hbj9zb1gec6wq4u/Isinor%20Permit%20Mission%20in%20Odyssey%202021-11-08%2019-52-41.mkv?dl=0

This is the procedure I followed, as I thought this would provide the most information:

1. Start recording
2. Click 'all missions'
3. Click the permit mission
4. Check the transaction tab (left view) to show no current missions
5. Check messages (top left view) to show last message received
6. Accept mission (get message and hear that I've received it too)
7. Check transaction tab (left view) again to show still no mission in log
8. Check messages (top left view) again to show the newly received permit issuing message
9. Check permits list (right panel) for permit (Isinor is on the bottom right)
10. End recording

I've not done anything else yet, because I wanted to see if the above was acceptable for your needs.  If it is, then I'll try the same procedure with another permit mission, but in Horizons instead.

**Comment by Tkael** [comment]
Thanks for the video. Process looks good.
From the video and particularly from the mission's flavor text, it looks like an intentional change.
Please replicate the process in Horizons and let me know if the flow is the same or different. o7

**Comment by Darkcyde13** [comment]
Sorry about the wait.  I've now had some time to try this out in Horizons, and I've found that it's exactly the same.  Here's the link to the video I made:
https://www.dropbox.com/s/t68dfnkqqg2ki34/Luyten%20347-14%20Permit%20Mission%20in%20Horizons%202021-11-13%2023-00-15.mkv?dl=0
Just as in Odyssey, the permit is issued as soon as you accept the mission.  It no longer appears in your missions transaction tab.  So both Horizons and Odyssey are exactly the same.
